find: `./fixtures/factory/stage/advertise_failure.i6pevu.helper': No such file or directory
find: `./fixtures/factory/stage/after_file_list.i6pevu.lst': No such file or directory
find: `./fixtures/factory/stage/after_file_list.i8egGb.lst': No such file or directory
find: `./fixtures/factory/stage/cat_consts.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/check_proxy.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/check_wn_drainstate.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/collector_setup.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/condor_bin_0.i6pevu.tgz': No such file or directory
find: `./fixtures/factory/stage/condor_bin_1.i6pevu.tgz': No such file or directory
find: `./fixtures/factory/stage/condor_bin_2.i6pevu.tgz': No such file or directory
find: `./fixtures/factory/stage/condor_config.check.i6pevu.include': No such file or directory
find: `./fixtures/factory/stage/condor_config.dedicated_starter.i6pevu.include': No such file or directory
find: `./fixtures/factory/stage/condor_config.i6pevu': No such file or directory
find: `./fixtures/factory/stage/condor_config.monitor.i6pevu.include': No such file or directory
find: `./fixtures/factory/stage/condor_config.multi_schedd.i6pevu.include': No such file or directory
find: `./fixtures/factory/stage/condor_platform_select.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/condor_startup.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/condor_vars.i6pdq1.lst': No such file or directory
find: `./fixtures/factory/stage/condor_vars.i8egGb.lst': No such file or directory
find: `./fixtures/factory/stage/constants.i6pdq1.cfg': No such file or directory
find: `./fixtures/factory/stage/constants.i8egGb.cfg': No such file or directory
find: `./fixtures/factory/stage/create_mapfile.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/create_temp_mapfile.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/description.i6pevu.cfg': No such file or directory
find: `./fixtures/factory/stage/description.i8egGb.cfg': No such file or directory
find: `./fixtures/factory/stage/error_augment.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/error_gen.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/file_list.i6pevu.lst': No such file or directory
find: `./fixtures/factory/stage/file_list.i8egGb.lst': No such file or directory
find: `./fixtures/factory/stage/gcb_setup.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/glexec_setup.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/glidein_cpus_setup.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/glidein_lib.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/glidein_memory_setup.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/glidein_sitewms_setup.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/grid-mapfile.i6pdq1': No such file or directory
find: `./fixtures/factory/stage/grid-mapfile.i8egGb': No such file or directory
find: `./fixtures/factory/stage/java_setup.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/mjf_setparams.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/parse_starterlog.i6pevu.awk': No such file or directory
find: `./fixtures/factory/stage/setup_network.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/setup_script.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/setup_x509.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/signature.i6pevu.sha1': No such file or directory
find: `./fixtures/factory/stage/signature.i8egGb.sha1': No such file or directory
find: `./fixtures/factory/stage/singularity_setup.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/smart_partitionable.i6pevu.sh': No such file or directory
find: `./fixtures/factory/stage/untar.i6pdq1.cfg': No such file or directory
find: `./fixtures/factory/stage/untar.i8egGb.cfg': No such file or directory
find: `./fixtures/factory/stage/validate_node.i6pevu.sh': No such file or directory
TESTING ==========> ./test_cleanupSupport.py

Running tests...
----------------------------------------------------------------------
Removed 10 files for 0.00MB.
.
----------------------------------------------------------------------
Ran 1 test in 0.025s

OK

Generating XML reports...
TESTING ==========> ./test_condorExe.py

Running tests...
----------------------------------------------------------------------
...
----------------------------------------------------------------------
Ran 3 tests in 2.798s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cWDictFile_DictFile.py

Running tests...
----------------------------------------------------------------------
....................
----------------------------------------------------------------------
Ran 20 tests in 0.063s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cWDictFile_DictFileTwoKeys.py

Running tests...
----------------------------------------------------------------------
.....................
----------------------------------------------------------------------
Ran 21 tests in 0.058s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cWDictFile_DictFileTwoKeys_SubClasses.py

Running tests...
----------------------------------------------------------------------
.....
----------------------------------------------------------------------
Ran 5 tests in 0.008s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cWDictFile_SHADicts.py

Running tests...
----------------------------------------------------------------------
.........
----------------------------------------------------------------------
Ran 9 tests in 0.016s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cWDictFile_validate_node.py

Running tests...
----------------------------------------------------------------------
...
----------------------------------------------------------------------
Ran 3 tests in 0.067s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cgWConsts.py

Running tests...
----------------------------------------------------------------------
.........
----------------------------------------------------------------------
Ran 9 tests in 0.010s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cgWDictFile.py

Running tests...
----------------------------------------------------------------------
SSS.....SSS...SSSSSSSSSSSS..SSSS.S....S.............S.......SSS...
----------------------------------------------------------------------
Ran 66 tests in 3.899s

OK (skipped=28)

Generating XML reports...
TESTING ==========> ./test_creation_lib_cgWParams.py

Running tests...
----------------------------------------------------------------------
..........S
----------------------------------------------------------------------
Ran 11 tests in 0.416s

OK (skipped=1)

Generating XML reports...
Coverage.py warning: Trace function changed, measurement is likely wrong: None (trace-changed)
TESTING ==========> ./test_creation_lib_cvWConsts.py

Running tests...
----------------------------------------------------------------------
.......
----------------------------------------------------------------------
Ran 7 tests in 0.011s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cvWParams.py

Running tests...
----------------------------------------------------------------------
...................
----------------------------------------------------------------------
Ran 19 tests in 0.631s

OK

Generating XML reports...
Coverage.py warning: Trace function changed, measurement is likely wrong: None (trace-changed)
TESTING ==========> ./test_creation_lib_factoryXmlConfig.py

Running tests...
----------------------------------------------------------------------
...........................
----------------------------------------------------------------------
Ran 27 tests in 6.934s

OK

Generating XML reports...
TESTING ==========> ./test_encodingSupport.py

Running tests...
----------------------------------------------------------------------
.....
----------------------------------------------------------------------
Ran 5 tests in 0.022s

OK

Generating XML reports...
TESTING ==========> ./test_factory_glideFactoryConfig.py

Running tests...
----------------------------------------------------------------------
...........................+++
.....................................................................................................+++
...
----------------------------------------------------------------------
Ran 14 tests in 0.759s

OK

Generating XML reports...
TESTING ==========> ./test_factory_glideFactoryEntry.py

Running tests...
----------------------------------------------------------------------
S.SS.......SS...SSS.S.SS.SSS.S.S.SSSSSSS
----------------------------------------------------------------------
Ran 40 tests in 2.856s

OK (skipped=23)

Generating XML reports...
TESTING ==========> ./test_factory_glideFactoryLib.py

Running tests...
----------------------------------------------------------------------
SSS.SSSSSSSSSSSSSSSS.........S.........SSSSSSSSSSSSSSS.SSSSSSSSSSSS.SS.
----------------------------------------------------------------------
Ran 71 tests in 0.096s

OK (skipped=49)

Generating XML reports...
TESTING ==========> ./test_frontend.py

Running tests...
----------------------------------------------------------------------
.........................Running glidein ids at ('submit.local', 'Site_Name2@v3_0@factory1', 'frontend@factory1') (total glideins: 3, total jobs 4, cluster matches: 4): glidein_5@cmswn005.local, glidein_2@cmswn002.local, glidein_1@cmswn006.local
Running glidein ids at ('submit.local', 'Site_Name1@v3_0@factory1', 'frontend@factory1') (total glideins: 1, total jobs 1, cluster matches: 1): glidein_1@cmswn001.local
Running glidein ids at ('submit.local', 'Site_Name3@v3_0@factory1', 'frontend@factory1') (total glideins: 0, total jobs 0, cluster matches: 0): 
Running glidein ids at ('submit.local', 'Site_Name2@v3_0@factory1', 'frontend@factory1') (total glideins: 0, total jobs 0, cluster matches: 0): 
Running glidein ids at ('submit.local', 'Site_Name1@v3_0@factory1', 'frontend@factory1') (total glideins: 0, total jobs 0, cluster matches: 0): 
Running glidein ids at ('submit.local', 'Site_Name3@v3_0@factory1', 'frontend@factory1') (total glideins: 0, total jobs 0, cluster matches: 0): 
Running glidein ids at ('submit.local', 'Site_Name2@v3_0@factory1', 'frontend@factory1') (total glideins: 1, total jobs 1, cluster matches: 1): glidein_2@cmswn002.local
Running glidein ids at ('submit.local', 'Site_Name1@v3_0@factory1', 'frontend@factory1') (total glideins: 1, total jobs 1, cluster matches: 1): glidein_1@cmswn001.local
Running glidein ids at ('submit.local', 'Site_Name3@v3_0@factory1', 'frontend@factory1') (total glideins: 0, total jobs 0, cluster matches: 0): 
........
----------------------------------------------------------------------
Ran 33 tests in 9.427s

OK

Generating XML reports...
TESTING ==========> ./test_frontend_element.py

Running tests...
----------------------------------------------------------------------
..
----------------------------------------------------------------------
Ran 2 tests in 0.092s

OK

Generating XML reports...
TESTING ==========> ./test_glideFactoryDowntimeLib.py

Running tests...
----------------------------------------------------------------------
.....
----------------------------------------------------------------------
Ran 5 tests in 0.113s

OK

Generating XML reports...
TESTING ==========> ./test_glideinFrontendPlugins.py

Running tests...
----------------------------------------------------------------------
.....
----------------------------------------------------------------------
Ran 5 tests in 0.151s

OK

Generating XML reports...
TESTING ==========> ./test_lib_encodingSupport.py

Running tests...
----------------------------------------------------------------------
....
----------------------------------------------------------------------
Ran 4 tests in 3.441s

OK

Generating XML reports...
TESTING ==========> ./test_lib_exprParser.py

Running tests...
----------------------------------------------------------------------
...
----------------------------------------------------------------------
Ran 3 tests in 0.336s

OK

Generating XML reports...
TESTING ==========> ./test_lib_fork.py

Running tests...
----------------------------------------------------------------------
.............
----------------------------------------------------------------------
Ran 13 tests in 72.379s

OK

Generating XML reports...
TESTING ==========> ./test_lib_pubCrypto.py

Running tests...
----------------------------------------------------------------------
......++++++
........++++++
............................++++++
...++++++
...........++++++
......++++++
.
----------------------------------------------------------------------
Ran 3 tests in 0.059s

OK

Generating XML reports...
TESTING ==========> ./test_lib_servicePerformance.py

Running tests...
----------------------------------------------------------------------
..........
----------------------------------------------------------------------
Ran 10 tests in 0.011s

OK

Generating XML reports...
TESTING ==========> ./test_lib_symCrypto.py

Running tests...
----------------------------------------------------------------------
.......S........
----------------------------------------------------------------------
Ran 16 tests in 8.156s

OK (skipped=1)

Generating XML reports...
TESTING ==========> ./test_lib_timeConversion.py

Running tests...
----------------------------------------------------------------------
.F.....FF.F..FF.
======================================================================
ERROR [0.087s]: test_extract_human (__main__.TestTimeFunctions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_lib_timeConversion.py", line 64, in test_extract_human
    self.assertEqual(float(now), float(extractHuman(human)))
AssertionError: 1518767040.0 != 1518745440.0

======================================================================
ERROR [0.007s]: test_get_human (__main__.TestTimeFunctions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_lib_timeConversion.py", line 59, in test_get_human
    self.assertEqual(human, getHuman(now))
AssertionError: 'Fri Feb 16 01:44:00 2018' != 'Fri Feb 16 07:44:00 2018'
- Fri Feb 16 01:44:00 2018
?             ^
+ Fri Feb 16 07:44:00 2018
?             ^


======================================================================
ERROR [0.009s]: test_get_is_o8601__local (__main__.TestTimeFunctions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_lib_timeConversion.py", line 75, in test_get_is_o8601__local
    self.assertEqual(iso_local, getISO8601_Local(now))
AssertionError: '2018-02-16T01:44:00-06:00' != '2018-02-16T07:44:00+00:00'
- 2018-02-16T01:44:00-06:00
?             ^      ^ ^
+ 2018-02-16T07:44:00+00:00
?             ^      ^ ^


======================================================================
ERROR [0.009s]: test_get_rf_c2822__local (__main__.TestTimeFunctions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_lib_timeConversion.py", line 104, in test_get_rf_c2822__local
    self.assertEqual(rfc_2822_local, getRFC2822_Local(now))
AssertionError: 'Fri, 16 Feb 2018 01:44:00 -0600' != 'Fri, 16 Feb 2018 07:44:00 +0000'
- Fri, 16 Feb 2018 01:44:00 -0600
?                   ^       ^ ^
+ Fri, 16 Feb 2018 07:44:00 +0000
?                   ^       ^ ^


======================================================================
ERROR [0.002s]: test_get_t_zval (__main__.TestTimeFunctions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_lib_timeConversion.py", line 127, in test_get_t_zval
    self.assertNotEqual(getTZval(now), getTZval(now_dst))
AssertionError: 0 == 0

======================================================================
ERROR [0.009s]: test_get_time_in_format (__main__.TestTimeFunctions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_lib_timeConversion.py", line 122, in test_get_time_in_format
    self.assertEqual(rfc_2822_local, get_time_in_format(now, time_format))
AssertionError: 'Fri, 16 Feb 2018 01:44:00 -0600' != 'Fri, 16 Feb 2018 07:44:00 -0600'
- Fri, 16 Feb 2018 01:44:00 -0600
?                   ^
+ Fri, 16 Feb 2018 07:44:00 -0600
?                   ^


----------------------------------------------------------------------
Ran 16 tests in 2.481s

FAILED (errors=6)

Generating XML reports...
Tue May  7 18:18:43 UTC 2019 ERROR: ./test_lib_timeConversion.py failed with non zero exit code (1)
TESTING ==========> ./test_lib_x509Support.py

Running tests...
----------------------------------------------------------------------
.
----------------------------------------------------------------------
Ran 1 test in 0.029s

OK

Generating XML reports...
TESTING ==========> ./test_lib_xmlParse.py

Running tests...
----------------------------------------------------------------------
.................
----------------------------------------------------------------------
Ran 17 tests in 0.084s

OK

Generating XML reports...
TESTING ==========> ./test_logSupport.py

Running tests...
----------------------------------------------------------------------
./test_logSupport.py:35: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.config = yaml.load(file(config_file, 'r'))
....
----------------------------------------------------------------------
Ran 4 tests in 565.793s

OK

Generating XML reports...
TESTING ==========> ./test_renew_proxy.py

Running tests...
----------------------------------------------------------------------
...
----------------------------------------------------------------------
Ran 3 tests in 0.004s

OK

Generating XML reports...
TESTING ==========> ./test_tarSupport.py

Running tests...
----------------------------------------------------------------------
..
----------------------------------------------------------------------
Ran 2 tests in 0.042s

OK

Generating XML reports...
TESTING ==========> ./test_creation_lib_cgWParamDict.py

Running tests...
----------------------------------------------------------------------
SSSSSS.......SSSSSSSS
----------------------------------------------------------------------
Ran 21 tests in 3.490s

OK (skipped=14)

Generating XML reports...
TESTING ==========> ./test_creation_lib_cvWParamDict.py

Running tests...
----------------------------------------------------------------------
SS..S.SSS....S.SSSSS
----------------------------------------------------------------------
Ran 20 tests in 1.574s

OK (skipped=12)

Generating XML reports...
TESTING ==========> ./test_util.py

Running tests...
----------------------------------------------------------------------
.
----------------------------------------------------------------------
Ran 1 test in 0.001s

OK

Generating XML reports...
